Source/JavaScriptCore/b3/B3Procedure.cpp

```diff
@@ -141,6 +141,8 @@ Value* Procedure::addConstant(Origin origin, Type type, uint64_t bits)
         return add<ConstFloatValue>(origin, bitwise_cast<float>(static_cast<int32_t>(bits)));
     case Double:
         return add<ConstDoubleValue>(origin, bitwise_cast<double>(bits));
+    case V128:
+        return add<ConstDoubleValue>(origin, bitwise_cast<double>(0xbadbeefl));
```

Comment:
What is the purpose of this code? It seems that it is adding a broken code.
1. Why not keep it unreachable assertion?
2. If we need to run this code right now, then we need the appropriate implementation.